pytorch_lightning/plugins/checkpoint/torch.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+from typing import Any, Callable, Dict, Optional, Union
+
+import pytorch_lightning as pl
+from pytorch_lightning.plugins.checkpoint.checkpoint import CheckpointIOPlugin
+from pytorch_lightning.utilities import rank_zero_warn
+from pytorch_lightning.utilities.cloud_io import atomic_save
+from pytorch_lightning.utilities.cloud_io import load as pl_load
+
+
+class TorchCheckpointIOPlugin(CheckpointIOPlugin):
+    def save_checkpoint(
+        self, checkpoint: Dict[str, Any], path: Union[str, Path], storage_options: Optional[Any] = None
```

Comment:
storage_options is unused. should it? or did mypy force you to add it?

Reply:
Complains if I don't add it to the signature